src/entity/builtin_entity.rs

```diff
@@ -430,6 +432,54 @@ impl BuiltinEntityKind {
         }
     }
 
+    fn pt_examples(&self) -> &'static [&'static str] {
+        match *self {
+            BuiltinEntityKind::AmountOfMoney => &[
+                "$10",
+                "15€",
+                "cinco euros",
+                "16,65 €",
+                "dois euros e cinco centavos",
+                "dez libras esterlinas",
+                "845584 francos suíços",
+            ],
+            BuiltinEntityKind::Duration => &[
+                "1h",
+                "3 meses",
+                "dez minutos",
+                "meia hora",
+                "oito anos e dois semanas",
+                "um ano quatro semanas e tres horas",
+            ],
+            BuiltinEntityKind::Number => &[
+                "2001",
```

Comment:
Can we add examples of with 1, 2 and 3 tokens

Reply:
Not for numbers, no resolution rules for verbalized numbers yet